sky/tools/create_full_ios_framework.py

```diff
@@ -192,12 +233,19 @@ def zip_archive(dst):
       'Flutter.xcframework',
       'entitlements.txt',
       'without_entitlements.txt',
+      'extension_safe/Flutter.xcframework',
```

Comment:
In the original PR, `extension_safe/Flutter.framework` and `extension/sim/` were mistakenly added to this list. Only `Flutter.xcframework` should be uploaded, just like the outside folder, see line 233 above.

Reply:
:+1: